src/dotnet/commands/dotnet-install/PackageToProjectFileAdder.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.ExecutablePackageObtainer;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.Cli
+{
+
+    internal class PackageToProjectFileAdder : ICanAddPackageToProjectFile
```

Comment:
Please name the interface like the class with I prefix:  IPackageToProjectFileAdder.

Ditto for the other interface.

ICanXxx is awkward and it's helpful IMHO when there are 1:1 names for interface to impl for common case.

Reply:
Fixed